frames/src/main/java/com/checkout/frames/screen/paymentdetails/PaymentDetailsScreen.kt

```diff
@@ -0,0 +1,42 @@
+package com.checkout.frames.screen.paymentdetails
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.checkout.frames.di.base.Injector
+import com.checkout.frames.style.screen.PaymentDetailsStyle
+import com.checkout.frames.utils.constants.PaymentDetailsScreenConstants
+
+@SuppressWarnings("MagicNumber")
+@Composable
+internal fun PaymentDetailsScreen(
+    style: PaymentDetailsStyle,
+    injector: Injector
+) {
+    val viewModel: PaymentDetailsViewModel = viewModel(
+        factory = PaymentDetailsViewModel.Factory(injector, style)
+    )
+
+    Column(modifier = Modifier.padding(PaymentDetailsScreenConstants.padding.dp)) {
+
+        // TODO: Replace with header component
+        Text(
+            text = "Payment details",
```

Comment:
Can we put it into strings.xml file?

Reply:
It's a dummy header, new header will be done appropriately in another ticket.